### DISCUSSION TITLE
numeric type field

### DISCUSSION THREAD

**Discussion by joaopaulozz** [RensTillmann/super-forms#71] [Q&A] [answered]
Hi,
I need a field on the form that only accepts numbers, which one can I use?
![Capturar](https://user-images.githubusercontent.com/69809959/191002657-0ab5c456-769c-47ab-80d9-5fd20e7a26f4.PNG)

**Comment by RensTillmann** [comment, marked as answer]
Hi, you can just add "Text field" and set it's Type to: [number]:

![image](https://user-images.githubusercontent.com/16322803/191064984-e55e1062-5792-444b-8a2e-4e3d85b4e72d.png)

![image](https://user-images.githubusercontent.com/16322803/191064773-83b5ed7f-443b-4ba5-82be-6979f720a2fc.png)

You can also use the quantity field depending on your needs:

![image](https://user-images.githubusercontent.com/16322803/191064873-dfef43ec-ff10-4c69-8409-bd4a3ed4c40c.png)

![image](https://user-images.githubusercontent.com/16322803/191064903-054a5e71-f2e4-4a28-8b66-aa1b587672a8.png)

**Reply by joaopaulozz** [reply]
Great!!!
Thank you
